preprocess-and-train.py

```diff
@@ -11,6 +11,8 @@
 # not a lot of images will be there as well.
 def preprocess(img):
     # your code goes here
+     img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY) #Converts images to gray
+    img = cv2.resize(img,(225,225)) #Resizes images to 225 x 225
```

Comment:
what other preprocessing techniques did you try?
and what were the results of those techniques? @siddharth25pandey

Reply:
We have not tried any other preprocessing techniques besides color conversion to Gray and resizing. As others have converted to RGB instead of gray, we will have to test the accuracy for that as well and compare.

Reply:
Okay, please explore other pre-processing options as well, just to learn about them. For this task, I would say this is enough so I will be approving the review.